src/js/common.js

```diff
@@ -1,12 +1,7 @@
-function trace(text) {
-  // This function is used for logging.
-  if (text[text.length - 1] === '\n') {
-    text = text.substring(0, text.length - 1);
-  }
-  if (window.performance) {
-    var now = (window.performance.now() / 1000).toFixed(3);
```

Comment:
This means that we crash on Chrome versions before performance.now being added.
Does that affect "min supported version" for the demos? Where is that documented?
I know we have a check in adapter.js on this, but I'm not sure we ever do anything with it.